### ISSUE TITLE
The shape of ball icon in maze game is inconsistent.

### ISSUE THREAD

**Issue by AnandChourasia007** [llaske/sugarizer#981]
This happens due to the way the icon was designed. One way to handle this will be to replace the current icon by a solid that always maintains its shape.

https://user-images.githubusercontent.com/89119829/155840319-c886b854-8f40-4a88-8e0d-7a722af1439f.mp4

**Comment by ankur12-1610** [comment]
I would like to work on this issue.

**Comment by AnandChourasia007** [comment]
Sorry @ankur12-1610 , but I have already almost completed my code for this issue. I asked here just for this issue to be approved.

**Comment by AnandChourasia007** [comment]
Yes,using a solid icon won't allow for the color change.I think I will close this issue and try finding amore non trivial bug or feature.Thanks
